cmd/snap/inhibit.go

```diff
@@ -80,9 +80,7 @@ func inhibitMessage(snapName string, hint runinhibit.Hint) string {
 }
 
 var isGraphicalSession = func() bool {
-	// TODO: uncomment once there is a proper UX review
-	//return os.Getenv("DISPLAY") != "" || os.Getenv("WAYLAND_DISPLAY") != ""
-	return false
+	return osGetenv("DISPLAY") != "" || osGetenv("WAYLAND_DISPLAY") != ""
```

Comment:
this seems premature, did we do the UX review mentioned in the TODO?

Reply:
I was a bit confused what was a UX review, I reverted the changes there. Thanks!